### ISSUE TITLE
Incorrect Qt Version Required to Build on Linux (Ubuntu)

### ISSUE THREAD

**Issue by breadsax** [BeamMW/beam-ui#638]
The SFTex.qml file in the beam-ui/ui/view/controls directory wants to:

import QtQuick 2.15 (this was changed on March 24th 2021 it appears)

That means that Qt 5.15 should be installed. I tried patching the code to import QtQuick2.12, but that results in the Wallet tab being blank with a blue background, even though all other tabs work as expected.

Why are the build instructions not updated for this? 

Also, 2.15 (Qt 5.15) is not supported with Ubuntu apt-get repositories, even in Ubuntu 20.10 only 2.14 (5.14) is supported with apt-get repositories.

Would it work with QtQuick 2.14 (Qt 5.14) as expected or am I no longer able to compile without installing a different distro? If I force install Qt 5.15 I'm afraid of breaking other things.

Thank you!

**Comment by breadsax** [comment]
@anatolse Update:  I made some progress. I turns out that Ubuntu 21.04 has Qt 5.15 in official repository, so I compiled with that and ran into only one small issue.

All of the icons are missing and there is just a black box. All of the Beam icons, the icons for each tab, etc. are showing a black box only.

Do you have any idea what would cause this? I installed all dependencies listed for Ubuntu 19.10 and also those listed in the .travis_no_build.yaml file just in case. 

Is there a different dependency used by the Beam dev team that is not included in the build instructions? It would be helpful if someone could take a look and I can help update the build instructions together with you.

Thank you!

**Comment by breadsax** [comment]
@anatolse and Beam Team, I have solved this! 

You do indeed need Qt 5.15 for this, but Ubuntu can upgrade to 20.10 for this officially. You updated some of the build instructions to suggest Qt 5.15 which is great. Will need to adjust some still though.

There is an upgrade error for some Ubuntu and Kubuntu users which causes these black boxes. See attached reference articles. Solved by running:

apt install --reinstall --allow-downgrades -y libqt5quick5

https://askubuntu.com/questions/1288506/black-icons-after-reinstalling-kubuntu-desktop

https://www.reddit.com/r/kde/comments/jhqbnz/kde_plasma_rendering_problem_black_squares/
